### PR TITLE
[@theia/console] Remove workaround that is no longer necessary

### PR DESCRIPTION
**Pull request by RDIL**
Signed-off-by: Reece Dunham <me@rdil.rocks>

#### What it does

My PR in Anser (https://github.com/IonicaBizau/anser/pull/60) landed, meaning that the misleading TypeScript types no longer exist, so we can just use the proper way of importing Anser.

#### How to test

I've verified that the Theia console still works.

#### Review checklist

- [X] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/console/compile.tsconfig.json [hunk 1]
```diff
@@ -16,4 +17,4 @@
       "path": "../monaco/compile.tsconfig.json"
     }
   ]
-}
+}
```
Comment (RDIL) [inline packages/console/compile.tsconfig.json:20]: When working in Gitpod, Theia quite literally wouldn't let me restore this new line for some reason

File: packages/console/compile.tsconfig.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "composite": true,
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
+    "esModuleInterop": true
```
Comment (vince-fugnitto) [inline packages/console/compile.tsconfig.json:7]: Why is the following change necessary?

File: packages/console/compile.tsconfig.json (hunk 1 above)
Comment (vince-fugnitto) [inline packages/console/compile.tsconfig.json:20]: Please revert.

File: packages/console/package.json [hunk 3]
```diff
@@ -44,4 +44,4 @@
   "nyc": {
     "extends": "../../configs/nyc.json"
   }
-}
+}
```
Comment (vince-fugnitto) [inline packages/console/package.json:47]: Please revert.

File: packages/console/compile.tsconfig.json (hunk 2 above)
Comment (RDIL) [reply packages/console/compile.tsconfig.json:7]: TSC complains that you can't use default imports without this enabled.

File: packages/console/src/browser/ansi-console-item.tsx [hunk 4]
```diff
@@ -17,10 +17,7 @@
 import * as React from 'react';
 import { ConsoleItem } from './console-session';
 import { Severity } from '@theia/core/lib/common/severity';
-
-// TODO: Remove this workaround once issue is fixed by anser.
-// REF: https://github.com/IonicaBizau/anser/issues/52
-const { ansiToHtml } = (require('anser') as import('anser').default);
+import Anser from 'anser';
```
Comment (marechal-p) [inline packages/console/src/browser/ansi-console-item.tsx:20]: `import Anser = require('anser')` is better in this case, no more need for `esModuleInterop`.

Comment (RDIL) [reply packages/console/src/browser/ansi-console-item.tsx]: Fixed.

**Review by marechal-p — APPROVED** [review]
LGTM, CI failure seems unrelated.

**Pull request merged**